scripts/merge-and-build.sh

```diff
@@ -74,35 +74,6 @@ if [ "${OSE_VERSION}" != "3.2" ] ; then
   ssh-add ${HOME}/.ssh/id_rsa
 fi # End check if we are version 3.2
 
-function sanity_check() {
```

Comment:
Here is the error I am getting back from the editor now:
```
$ GIT_SEQUENCE_EDITOR=../aos-cd-jobs/scripts/rebase.py git rebase -i public/release-1.5 
Commit `Change image build and drop this cluster up flag` (c122009) invalid!
Could not execute editor
```
I will update the error messages but at this stage I don't see breaking out the validation as a big win.